contents/docs/developing-locally.md

```diff
@@ -21,15 +21,15 @@ docker-compose -f docker-compose.dev.yml up
 
 ## Using virtualenv
 
-1. Make sure you have Python 3 installed `python3 --version`. [pyenv](https://github.com/pyenv/pyenv) is recommended to manage multiple Python versions and make sure you don't use the system version.
-2. Make sure you have [Redis installed](https://redis.io/download) and running.
+1. Make sure you have Python 3.8 installed `python3 --version`. [pyenv](https://github.com/pyenv/pyenv) is recommended to manage multiple Python versions and make sure you don't use the system version.
+2. Make sure you have [Redis installed](https://redis.io/download) and running (v5 should work).
```

Comment:
Just as a general point, we should aim to be assertive when writing up development steps. "Should work" doesn't give me confidence in this "tutorial". Maybe "v5 or above"? Also, I'm giving this feedback because it was given to me a while back, and it makes a lot of sense in my view.